voc/risk_impact.go

```diff
@@ -26,5 +28,7 @@
 package voc
 
 type RiskImpact struct {
```

Comment:
delete

Reply:
I delete the RiskImpact in the Ontology.